chrome/content/coniks/context.js

```diff
@@ -0,0 +1,23 @@
+this.EXPORTED_SYMBOLS = ["coniksContext"];
+
+function coniksContext(opts) {
```

Comment:
The context could also contain a field for the current TB (if available) for the account.